src/makeCalculator.js

```diff
@@ -4,7 +4,36 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
+    result: 0,
+
+    add(x) {
+      calc.result += x;
+    },
+    subtract(x) {
+      calc.result -= x;
+    },
+    multiply(x) {
+      calc.result *= x;
+    },
+    divide(x) {
+      calc.result /= x;
```

Comment:
There is a potential issue in the `divide` method. If `x` is zero, this will result in a division by zero, which could lead to unexpected results or errors. Consider adding a check to handle division by zero cases.